spec/javascripts/components/stories/history_spec.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import History, {
+  Changes,
+  Header
+} from '../../../../app/assets/javascripts/components/stories/History';
+import HistoryFactory from '../../support/factories/historyFactory';
+
+describe('<History />', () => {
+  const wrapper = mount(<History history={HistoryFactory()} />);
```

Comment:
Avoid creating variables outside like that. Some reasons why:

- it makes you share state between tests
- even if you skip all the tests inside this describe (for example, to run only the tests from another file), this mount will still be called

Prefer to do it like this:

```suggestion
  let wrapper;
  
  beforeEach(() => {
    wrapper = mount(<History history={HistoryFactory()} />);
  });
```

Reply:
Thanks for the advice